src/thunderbots/software/ai/navigator/util.cpp

```diff
@@ -5,3 +5,19 @@ double calculateTransitionSpeedBetweenSegments(const Point &p1, const Point &p2,
 {
     return final_speed * (p2 - p1).norm().project((p3 - p2).norm()).len();
 }
+
+std::vector<MovePrimitive> convertToMovePrimitives(unsigned int robot_id,
+                                                   const std::vector<Point> &points)
+{
+    std::vector<MovePrimitive> movePrimitives;
+    movePrimitives.reserve(points.size());
```

Comment:
you can do this in the constructor

Reply:
What constructor? This is just a utility function

Reply:
I meant the vector constructor but it turns out that doesn't work

Reply:
resolved in slack, keeping "reserve"